include/openssl/crmferr.h

```diff
@@ -2,68 +2,67 @@
  * Generated by util/mkerr.pl DO NOT EDIT
  * Copyright 1995-2019 The OpenSSL Project Authors. All Rights Reserved.
  *
- * Licensed under the OpenSSL license (the "License").  You may not use
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_CRMFERR_H
-# define HEADER_CRMFERR_H
+#ifndef OSSL_HEADER_CRMFERR_H
+# define OSSL_HEADER_CRMFERR_H
 
-# include <openssl/opensslconf.h>
-
-# ifndef OPENSSL_NO_CMP
```

Comment:
and, why are these two lines removed?

Reply:
This issue also went away  by using #8897. The line now reads
```
# ifndef OPENSSL_NO_CRMF
```